app/code/community/Phoenix/CashOnDelivery/etc/system.xml

```diff
@@ -48,15 +48,15 @@
                     <show_in_website>1</show_in_website>
                     <show_in_store>1</show_in_store>
                     <fields>
-                        <active translate="label">
+                        <enable translate="label">
```

Comment:
Mage_Payment_Model_Method_Abstract::isAvailable() checks for "active" config path, not "enable".